README.md

```diff
@@ -1,4 +1,14 @@
 # Smart-Mirror
 Raspberry powered mirror which can display the news, weather, and time.
 
+## Installation and Updating
+Installation is simple as running the following command.
+
+> sudo pip install git+https://github.com/HackerHouseYT/Smart-Mirror.git
+
+## Running
+To run the application run the following command in this folder
+> python smartmirror.py
```

Comment:
Its up to @atainter, I do like not editing config files and run everything from CLI. Perhaps someone could add something like this.

> pyhton smartmirror.py --token TOKEN_KEY